### ISSUE TITLE
[qtinterfaceframework] Build error on x64-windows

### ISSUE THREAD

**Issue by Bunie89** [microsoft/vcpkg#35963]
(Windows 11 23H2, Sorry if i didn't do this right lol)

Package: qtinterfaceframework:x64-windows@6.6.1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.39.33321.0
-    vcpkg-tool version: 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e
    vcpkg-scripts version: 839722725 2023-12-29 (3 days ago)

**To Reproduce**

`vcpkg install qt:x64-windows`
**Failure logs**

```
-- Using cached D:/Apps/vcpkg/downloads/qtinterfaceframework-a2085374e38a135eaacfe7533888f0a361d4c721.tar.gz
-- Cleaning sources at D:/Apps/vcpkg/buildtrees/qtinterfaceframework/src/a361d4c721-2c693f67de.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/Apps/vcpkg/downloads/qtinterfaceframework-a2085374e38a135eaacfe7533888f0a361d4c721.tar.gz
-- Applying patch fix-taglib-search.patch
-- Using source at D:/Apps/vcpkg/buildtrees/qtinterfaceframework/src/a361d4c721-2c693f67de.clean
-- Using cached mingw-w64-x86_64-pkgconf-1~2.1.0-1-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.9-3-x86_64.pkg.tar.zst.
-- Using msys root at D:/Apps/vcpkg/downloads/tools/msys2/023cdb3ca06f77f2
-- Using cached pypa-get-pip-38e54e5de07c66e875c11a1ebbdb938854625dd8.tar.gz.
-- Cleaning sources at D:/Apps/vcpkg/buildtrees/qtinterfaceframework/src/8854625dd8-861bd167bd.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/Apps/vcpkg/downloads/pypa-get-pip-38e54e5de07c66e875c11a1ebbdb938854625dd8.tar.gz
-- Using source at D:/Apps/vcpkg/buildtrees/qtinterfaceframework/src/8854625dd8-861bd167bd.clean
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: D:/Apps/vcpkg/downloads/tools/python/python-3.11.5-x64/python.exe D:/Apps/vcpkg/buildtrees/qtinterfaceframework/src/8854625dd8-861bd167bd.clean/public/get-pip.py --no-warn-script-location
    Working Directory: D:/Apps/vcpkg/buildtrees/qtinterfaceframework
    Error code: 1
    See logs for more information:
      D:\Apps\vcpkg\buildtrees\qtinterfaceframework\get-pip-x64-windows-err.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-get-python-packages/x_vcpkg_get_python_packages.cmake:48 (vcpkg_execute_required_process)
  ports/qtinterfaceframework/portfile.cmake:22 (x_vcpkg_get_python_packages)
  scripts/ports.cmake:170 (include)



```
<details><summary>D:\Apps\vcpkg\buildtrees\qtinterfaceframework\get-pip-x64-windows-err.log</summary>

```
Traceback (most recent call last):
  File "D:\Apps\vcpkg\buildtrees\qtinterfaceframework\src\8854625dd8-861bd167bd.clean\public\get-pip.py", line 33324, in <module>
    main()
  File "D:\Apps\vcpkg\buildtrees\qtinterfaceframework\src\8854625dd8-861bd167bd.clean\public\get-pip.py", line 135, in main
    bootstrap(tmpdir=tmpdir)
  File "D:\Apps\vcpkg\buildtrees\qtinterfaceframework\src\8854625dd8-861bd167bd.clean\public\get-pip.py", line 111, in bootstrap
    monkeypatch_for_cert(tmpdir)
  File "D:\Apps\vcpkg\buildtrees\qtinterfaceframework\src\8854625dd8-861bd167bd.clean\public\get-pip.py", line 92, in monkeypatch_for_cert
    from pip._internal.commands.install import InstallCommand
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\commands\__init__.py", line 9, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\cli\base_command.py", line 15, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\cli\cmdoptions.py", line 23, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\cli\parser.py", line 12, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\configuration.py", line 24, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_internal\utils\appdirs.py", line 13, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_vendor\platformdirs\__init__.py", line 33, in <module>
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_vendor\platformdirs\__init__.py", line 29, in _set_platform_dir_class
  File "importlib\__init__.py", line 126, in import_module
  File "C:\Users\Bunie\AppData\Local\Temp\tmp8a5q44us\pip.zip\pip\_vendor\platformdirs\windows.py", line 3, in <module>
  File "ctypes\__init__.py", line 8, in <module>
ImportError: DLL load failed while importing _ctypes: The specified module could not be found.
```
</details>

**Comment by Neumann-A** [comment]
By any chance: Do you have a python installed system wide?

**Comment by Bunie89** [comment]
Gonna be honest, i have no idea.

**Comment by Neumann-A** [comment]
go into a command prompt and type python. If it starts a subshell you have a python installed system wide. If it starts the microsoft appstore you have it at least not on the path.

**Comment by Bunie89** [comment]
It does indeed start a subshell for Python 3.12.0

**Comment by LilyWangLL** [comment]
Duplicated of https://github.com/microsoft/vcpkg/issues/33346.

**Comment by Neumann-A** [comment]
duplicate of #34746

**Comment by LilyWangLL** [comment]
Closed as duplicate of #34746, this issue has been fixed in PR #35990.
